File: day26.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner sc = new Scanner(System.in);
        int d1 = sc.nextInt();
        int m1 = sc.nextInt();
        int y1 = sc.nextInt();
        int d2 = sc.nextInt();
        int m2 = sc.nextInt();
        int y2 = sc.nextInt();
        int sum=0;
        int numberDays= (d1-d2) + (m1-m2)*30 + (y1 -y2) *365;
        if(numberDays > 365 || y1 > y2){
            sum=10000;
        }
        else if(numberDays > 30){
            sum = (m1-m2) * 500;
        }
        else if(numberDays >=1){
            sum = (d1 - d2) * 15 ;
        }
        else{
            sum=0;
        }

        System.out.println(sum);
    }
}

File: day26.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
d1, m1, y1 = map(int, input().strip().split())
d2, m2, y2 = map(int, input().strip().split())
sum=0
numberDays= (d1-d2) + (m1-m2)*30 + (y1 -y2) *365
if numberDays > 365 or y1 > y2:
    sum=10000
elif numberDays > 30:
     sum = (m1-m2) * 500
elif numberDays >=1:
     sum = (d1 - d2) * 15
else:
    sum=0

print(sum)
